love/src/components/ESS/AuxTel/AuxTelESS.container.jsx

```diff
@@ -99,88 +100,157 @@ const getGroupNames = (salindexList, option) => {
   return [...new Set(subscriptions)];
 };
 
+/**
+ * 
```

Comment:
Please add a description of the function, something like: "This function is used to parse dicts by..."